### ISSUE TITLE
Standardize the Dataset

### ISSUE THREAD

**Issue by MiguelSteph** [MiguelSteph/heart_disease_prediction#7]
- Fit and transform the training set
- Transform the validation set
- Transform the test set
